src/three/loaders/I3DMLoader.js

```diff
@@ -94,6 +93,7 @@ export class I3DMLoader extends I3DMLoaderBase {
 							}
 
 						} );
+						const rtcCenter = featureTable.getData( 'RTC_CENTER' );
```

Comment:
Lets define this as `RTC_CENTER` as the rest of the feature table values are and move it up above with the rest of the definitions to be consistent.